eZ/Publish/Core/FieldType/RichText/XSLTProcessorExtension.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace eZ\Publish\Core\FieldType\RichText;
+
+/**
+ * Helper functions available in XSL documents.
+ */
+class XSLTProcessorExtension
```

Comment:
Note to reviewer: Maybe this class should be called "XSLTProcessorFuntions" or in another way ?

Reply:
yes, that would be slightly better. but no strong opinion on it here.